.changeset/old-doors-kiss.md

```diff
@@ -0,0 +1,19 @@
+---
+'eslint-plugin-prettier': minor
+---
+
+Add recommended config for the flat config format.
+
+If you are using flat config, import the recommended config from `eslint-plugin-prettier/recommended`. Note that in contrast to the legacy recommended config, the flat recommended config does not add `eslint-config-prettier` automatically, consumers will need to explictly add that themselves.
+
+```js
+// eslint.config.js
+const eslintPluginPrettierRecommended = require('eslint-plugin-prettier/recommended');
+const eslintConfigPrettier = require('eslint-config-prettier');
+
+module.exports = [
+  // Any other config imports go at the top
+  eslintPluginPrettierRecommended,
+  eslintConfigPrettier,
```

Comment:
```suggestion
  eslintConfigPrettier,
  eslintPluginPrettierRecommended,
```

I would recommend to reverse the order, `eslintConfigPrettier` can be enabled globally, and `eslintPluginPrettierRecommended` may add/override some rules.

Reply:
I believe that what I've got currently is the preferred order for this.

[`eslint-config-prettier`'s docs](https://github.com/prettier/eslint-config-prettier?tab=readme-ov-file#installation) states that it should be the last item the list of configs. I don't want to be confusing by giving conflicting information.

There's never going to be a case where eslint-plugin-prettier shall turn back on rules that eslint-config-prettier has turned off.